test/endpoints.test.js

```diff
@@ -0,0 +1,43 @@
+const request = require('request');
+const expect = require('chai').expect;
+const server = require('./../server/server.js');
+const seeder = require('./../server/seeding-script/seeder');
+const mongoose = require('mongoose');
+
+describe('API Endpoints', () => {
+
+  test(`should make a GET request to the endpoint with a productId
+  and recieve back an array of original size images of the
+  product that belongs to the productId`, done => {
+
+    seeder.seedData((err, data) => {
+      let productId = 0;
+
+      request({
+        method: 'GET',
+        uri: `http://127.0.0.1:3000/images/org/${productId}`
```

Comment:
Not sure if this port 3000 will affect the proxy, since ProductSizes service uses port 3000.

Reply:
Just noting another place where port may need to be updated to 3004.